doc/lua_api.md

```diff
@@ -8225,7 +8225,9 @@ child will follow movement and rotation of that bone.
     * See `hud_set_flags` for a list of flags that can be toggled.
 * `hud_set_hotbar_itemcount(count)`: sets number of items in builtin hotbar
     * `count`: number of items, must be between `1` and `32`
+    * If `count` exceeds the `"main"` list size, the list size will be used instead.
 * `hud_get_hotbar_itemcount()`: returns number of visible items
+    * This value is not clamped by the `"main"` list size.
```

Comment:
so it's clamped when setting but not when getting?
doesn't seem useful to behave like this

Reply:
Fixed in a2e0345d4